proj1b/proj1b/Deque.java

```diff
@@ -0,0 +1,12 @@
+public interface Deque<T> {
+    void addFirst(T item);
+    void addLast(T item);
+    int size();
+    void printDeque();
+    T removeFirst();
+    T removeLast();
+    T get(int index);
+    default boolean isEmpty() {
+        return size() == 0;
+    }
+}
```

Comment:
Looks good. It would look more professional if Javadoc comments are added so that one could generate API documentation via `javadoc` command. See https://www.baeldung.com/javadoc.

```
/**
 * Deque (usually pronounced like “deck”) is an irregular acronym of double-ended queue.
 * Double-ended queues are sequence containers with dynamic sizes that can be expanded 
 * or contracted on both ends (either its front or its back).
 **/
public interface Deque<T> {
    /**
     * Adds an item of type T to the front of the deque.
     * @param item: item to add
     **/
    void addFirst(T item);
```